lib/roku/index.js

```diff
@@ -0,0 +1,53 @@
+var assert = require('assert');
+var https = require('../https');
+
+var apiUrls = {
+	production: 'https://apipub.roku.com/listen/transaction-service.svc/validate-transaction'
+};
+
+exports.verifyPayment = function (payment, cb) {
+	try {
+		assert.equal(typeof payment.devToken, 'string', 'Developer ID must be a string');
+		assert.equal(typeof payment.receipt, 'string', 'Receipt must be a string');
+		assert.equal(payment.receipt.match(/[\d|\w]/g).length, 32, 'Receipt must contain 32 digits');
```

Comment:
\w is alphanum (so including digits), right?
If you wanna make sure that `payment.receipt` is exactly 32 characters, you don't wanna do a length check on the output of `match`. You wanna do a length check in the regexp: eg. `/^[0-9]{32}$/`